tests/kernel/blktests.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Block device layer tests
+# Maintainer: Michael Moese <mmoese@suse.de>, Sebastian Chlad <schlad@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+#use utils 'zypper_call';
+#use power_action_utils 'power_action';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    #below variable exposes blktests options to the openQA testsuite
+    #definition, so that it allows flexible ways of re-runing the tests
+    my $quick   = get_required_var('blk_QUICK');
+    my $exclude = get_required_var('blk_EXCLUDE');
+    my $config  = get_required_var('blk_CONFIG');
+    my $tests   = get_required_var('blk_TESTS');
+    my $device  = get_required_var('blk_DEVICE_ONLY');
+
```

Comment:
Maybe you should specify the group as well via a variable? This way, the same code could execute all of blktests without the need to be changed?

Reply:
actually this is meant for those:
my $tests   = get_required_var('blk_TESTS');

but yeah sure. The code is just now allowing me to run test locally. It needs re-work definitely.

Reply:
Ah I see, that's fine then.